docs/index.md

```diff
@@ -1,12 +1,12 @@
 # StashCache
 
-This documenation serve to Operations team and Site Admins who manage StashCache nodes (origin or cache server; or redirectors). Main functionality of StashCache is based on [XRootD](http://xrootd.org) technology, there are other components involved to function StashCache properly, though. E.g. if you are a user of **"stashcp"** you want to follow [this](https://support.opensciencegrid.org/support/solutions/articles/12000002775-transferring-data-with-stashcache) document instead.
+This documenation serves to Operations team and Site Admins who manage StashCache nodes (origin or cache server; or redirectors). Main functionality of StashCache is based on [XRootD](http://xrootd.org) technology, there are other components involved to function StashCache properly, though. E.g. if you are a user of **"stashcp"** you want to follow [this](https://support.opensciencegrid.org/support/solutions/articles/12000002775-transferring-data-with-stashcache) document instead.
 
 ---
 
 StashCache operations consist of the following documents:
 
 * [StashCache redirectors](https://confluence.grid.iu.edu/display/STAS/Installing+an+XRootD+StashCache+Federation+Redirector): Redirectors are managed by GOC Operations team at Indiana University, redirectors provides federated access to the data across origins.
-* [StashCache origins](https://github.com/opensciencegrid/StashCache/docs/admin): Managed by an organization, **"origin"** is data server to host files locally and serve them to users upon transfer request (via [stashcp](https://support.opensciencegrid.org/support/solutions/articles/12000002775-transferring-data-with-stashcach)) unless data are already cached.
-* [StashCache caches](https://github.com/opensciencegrid/StashCache/docs/admin): Managed by an organization, **"cache"** server to keep data cached and immediately available (via [stashcp](https://support.opensciencegrid.org/support/solutions/articles/12000002775-transferring-data-with-stashcach))within Stash federation (without re-transfering from "origin").
+* [StashCache origins](ihttps://opensciencegrid.github.io/StashCache/admin/install/): Managed by an organization, **"origin"** is data server to host files locally and serve them to users upon transfer request (via [stashcp](https://support.opensciencegrid.org/support/solutions/articles/12000002775-transferring-data-with-stashcach)) unless data are already cached.
```

Comment:
Misspell, I see you have an 'i' before the URL.

Also, for internal links (links within the same docs) you can follow this guide: http://www.mkdocs.org/user-guide/writing-your-docs/#linking-documents